lib/java-rpc-proto.gradle

```diff
@@ -52,6 +52,14 @@ configure(projectsWithFlags('java')) {
                             artifact = artifactName
                         }
                     }
+
+                    if (project.ext.hasFlag('akka-grpc')) {
```

Comment:
Should we use `akka-grpc_2.12` and add `akka-grpc_2.13`?

Reply:
That's a good suggestion. Fixed. 😉